tests/impl.h

```diff
@@ -134,6 +134,7 @@ enum InstructionTest {
     IT_MM_ADDS_EPI16,
     IT_MM_MAX_EPI16,
     IT_MM_SUBS_EPU16,
+    IT_MM_CMPLT_EPI16,
```

Comment:
At the comment `// Unit test implemented and verified as fully working` at the end.

Reply:
done